modules/deploy/pages/deployment-option/self-hosted/manual/linux-system-tuning.adoc

```diff
@@ -0,0 +1,28 @@
+
+== User-triggered hardware tuning
+
+Users invoke the Redpanda autotuner using the `rpk redpanda tune` command. This command may optimize both native Linux and Kubernetes nodes. It analyzes the hardware configuration of the worker node and sets appropriate kernel options. You can choose to set certain hardware parameters using `ulimit` or other system tools while using the autotuner to manage others.
+
+=== Using the autotuner
+
+The Redpanda autotuner has a large array of hardware tuners to choose from. You may choose to either execute all available tuners, or to apply only a subset of them. This is particularly important when you wish to maintain specific configurations and prevent Redpanda from altering these settings.
+
+Redpanda recommends running the autotuner as part of your production deployment workflows. This ensures any hardware configurations or updates to the autotuner itself are incorporated quickly and efficiently into your system.
+
+For the full list of available tuners, refer to the xref:reference:rpk/rpk-redpanda/rpk-redpanda-tune.adoc[`rpk redpanda tune` guide]. The autotuner is also available for managing Kubernetes nodes, but note that changes to the Linux kernel for running Kubernetes images are not persisted. For more information on Kubernetes configuration and execution, refer to xref:deploy:deployment-option/self-hosted/kubernetes/k-tune-workers.adoc[Kubernetes worker node tuning].
+
+== Automated software configurations
+
+Redpanda makes use of systemd to manage operating system and software-based limits, such as memlocks, file handle limits, core limits, and certain distribution-specific configurations. These help ensure the system executes stably and efficiently. Some key examples of automated configurations include:
+
+* Using systemd also offers some support to detect hung services and infinite looping scenarios. This helps ensure you don't have blocking services or processes that may interfere with Redpanda's stream processing.
+* Redpanda configures asynchronus input/output interfaces and scheduler affinity. This is particularly critical when you rely on real-time processing of data streams for your applications.
+* Using systemd slices (see https://www.scylladb.com/2019/09/25/isolating-workloads-with-systemd-slices/[this blog post] for more details) allows Redpanda to most efficiently use system resources while isolating tasks for scalability and performance reasons.
+
+== Monitoring configurations
+
+While Redpanda provides numerous metrics, the configurations applied using the autotuner and systemd are not exposed internally. You should continue monitoring your system health through standard means. For example, the systemd configuration sets the file handle limit to 800,000 references. If you are concerned about approaching this limit, you should monitor it through standard system monitoring, such as the use of the `lsof` utility.
```

Comment:
@deniscoady what do you think about this statement wrt overall monitoring approach.  Does this really make sense to recommend?

Also, are these configs in the diag bundle?

Wondering if its worth having this paragraph

Reply:
@greg-szrama Did you get the feedback/guidance from @deniscoady ?